samples/interactive-tutorials/product/remove-fulfillment-places.js

```diff
@@ -61,20 +61,20 @@ async function main(generatedProductId) {
 
     console.log('Remove fulfillment request:', request);
     // Run request
-    await retailClient.removeFulfillmentPlaces(request);
-
-    console.log('Waiting to complete remove operation..');
+    const [operation] = await retailClient.removeFulfillmentPlaces(request);
+    const response = await operation.promise();
+    console.log('Wait till the operation to be completed...');
+    console.log(response);
   };
 
   // Remove fulfillment places with current time
   console.log('Start remove fulfillment');
   await callRemoveFulfillmentPlaces();
-  await utils.delay(180000);
-
+  await utils.delay(60000);
```

Comment:
Will the customer running this sample know what the `utils` module represents?

Reply:
the utils library is a part of the project, it is placed in the nodejs-retail/samples/interactive-tutorials/setup/setup-cleanup.js, so user can use it without any issues